### PR TITLE
AM-2908: Invalid validation on User IDs to accommodate Old Format of …

### PR DESCRIPTION
**Pull request by prashanthkumarnb**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/AM-2908

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ *] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/judicialbooking/util/Constants.java [hunk 1]
```diff
@@ -16,8 +16,11 @@ private Constants() {
     public static final String TEXT_HYPHEN_PATTERN = "^[-a-zA-Z]*$";
     public static final String NUMBER_TEXT_HYPHEN_PATTERN = "^[-a-zA-Z0-9]*$";
     public static final String TEXT_PATTERN = "^[a-zA-Z]*$";
+
```
Comment (mikebrownccd) [inline src/main/java/uk/gov/hmcts/reform/judicialbooking/util/Constants.java:19]: unrequired new line added

**Review by mikebrownccd — DISMISSED** [review]
LGTM

**Review by mikebrownccd — APPROVED** [review]
LGTM

File: src/test/java/uk/gov/hmcts/reform/judicialbooking/util/ValidationUtilTest.java [hunk 2]
```diff
@@ -19,6 +19,11 @@
 
 class ValidationUtilTest {
 
+    static String uuidString1 = "5629957f-4dcd-40b8-a0b2-e64ff5898b28";
+    static String uuidString2 = "5629957f-4dcd-40b8-a0b2-e64ff5898b29";
```
Comment (abul-syed) [inline src/test/java/uk/gov/hmcts/reform/judicialbooking/util/ValidationUtilTest.java:23]: the string `uuidString2` is not used anywhere

Comment (prashanthkumarnb) [reply src/test/java/uk/gov/hmcts/reform/judicialbooking/util/ValidationUtilTest.java]: Addressed

**Review by mattnayler — APPROVED** [review]

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Review by abul-syed — APPROVED** [review]

**Pull request merged**